coinlib_flutter/windows/CMakeLists.txt

```diff
@@ -10,15 +10,17 @@ project(${PROJECT_NAME} LANGUAGES CXX)
 
 # Invoke the build for native code shared with the other target platforms.
 # This can be changed to accommodate different builds.
-add_subdirectory("${CMAKE_CURRENT_SOURCE_DIR}/../src" "${CMAKE_CURRENT_BINARY_DIR}/shared")
+# add_subdirectory("${CMAKE_CURRENT_SOURCE_DIR}/../src" "${CMAKE_CURRENT_BINARY_DIR}/shared")
+# TODO re-enable the above line if/when the CMake process is made Windows-compatible.  See also below.
 
 # List of absolute paths to libraries that should be bundled with the plugin.
 # This list could contain prebuilt libraries, or libraries created by an
 # external build triggered from this build file.
 set(coinlib_flutter_bundled_libraries
   # Defined in ../src/CMakeLists.txt.
   # This can be changed to accommodate different builds.
-  $<TARGET_FILE:coinlib_flutter>
-  ${CMAKE_BINARY_DIR}/../build/secp256k1.dll
+  # $<TARGET_FILE:coinlib_flutter>
+  # TODO re-enable the above line if/when the CMake process is made Windows-compatible.  See also above.
+  ${CMAKE_BINARY_DIR}/../secp256k1.dll
```

Comment:
If it's not possible to build the library using CMake as part of the flutter process, then it needs to pick up the library from the PWD where the build script was run. I'm not sure how to get that in the CMake flutter process.

Alternatively the binary could be hosted and downloaded.

Reply:
This is testing working as part of the coinlib_flutter example app flutter build process.

Reply:
If someone is using the package from pub.dev, the library wont be in the `CMAKE_BINARY_DIR` though right? The library needs to be picked up from the consumer's local build.

Reply:
I think CMAKE_BINARY_DIR  would be `<flutter_project>/build/windows/x64 (hence the `/../`).  I tested it using both path and git refs locally (uncommitted).  First I made sure the coinlib_flutter example works, then I made sure the whole setup works when included in another flutter project.  When I `dart run coinlib:build_wsl`, it spits the `secp256k1.dll` into `<flutter_project>/build/windows` and coinlib finds it there as well, so I think `CMAKE_BINARY_DIR` is valid here.

But it would probably be a good idea to put out a windows testing version as you mentioned before before adding it to a "mainstream" release to doublecheck that this is the case when using it from pub.dev.

Reply:
This conversation has been outdated by 91f0f7f0e66e6ded1f58715ba5d5653d2fbb9402 but still applies broadly.  I still think it's correct, do you have a suggested change?